### PR TITLE
Make version.py template black compliant

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: python/lsst/sconsUtils/builders.py [hunk 1]
```diff
@@ -602,20 +602,22 @@ def makeVersionModule(target, source, env):
...
-            outFile.write("%s = {\n" % (what))
-            for name, mod in env.dependencies.packages.items():
-                if mod is None:
-                    outFile.write("    '%s': None,\n" % name)
-                elif hasattr(mod.config, "version"):
-                    outFile.write("    '%s': '%s',\n" % (name, mod.config.version))
-                else:
-                    outFile.write("    '%s': 'unknown',\n" % name)
+            outFile.write("%s = {" % (what))
+            if len(env.dependencies.packages.items()) != 0:
```
Comment (timj) [inline python/lsst/sconsUtils/builders.py:606]: Is the .items and != 0 needed here? Would

```python
if env.dependencies.packages:
```
also work?

File: python/lsst/sconsUtils/builders.py [hunk 2]
```diff
@@ -573,15 +573,15 @@ def makeVersionModule(target, source, env):
             outFile.write("# -------- This file is automatically generated by LSST's sconsUtils -------- #\n")
 
             what = "__version__"
-            outFile.write("%s = '%s'\n" % (what, version))
```
Comment (timj) [inline python/lsst/sconsUtils/builders.py:576]: I'm fine with the single quotes being switched to the outer and the double quotes then not needing to be escaped.

f-strings can also be used these days.

**Review by timj — APPROVED** [review]
Couple of minor suggestions but I'm happy to make this file black compliant.

File: python/lsst/sconsUtils/builders.py (hunk 1 above)
Comment (couger01) [reply python/lsst/sconsUtils/builders.py]: Seems to work, thanks!

File: python/lsst/sconsUtils/builders.py (hunk 2 above)
Comment (couger01) [reply python/lsst/sconsUtils/builders.py:576]: Okay, done.

**Comment by couger01** [comment]
Okay requested changes were made. Please merge at your leisure.

**Comment by timj** [comment]
Would you mind making a jira ticket for your work and naming the branch tickets/XXX-NNNNN ?

**Pull request closed without merging**